crates/router/tests/connectors/adyen_uk_ui.rs

```diff
@@ -118,6 +118,35 @@ async fn should_make_adyen_twint_payment(driver: WebDriver) -> Result<(), WebDri
     Ok(())
 }
 
+async fn should_make_adyen_dana_payment(driver: WebDriver) -> Result<(), WebDriverError> {
```

Comment:
Can’t we have ui test for Alma?

Reply:
Alma requires OTP from mobile number for validation so can't be automated